test/cfg/std.c

```diff
@@ -518,7 +516,7 @@ void nullpointer(int value)
     wcstok(NULL,L"xyz",&pWcsUninit);
 
     strxfrm(0,"foo",0);
-    // TODO: error message (#6306 and http://trac.cppcheck.net/changeset/d11eb4931aea51cf2cb74faccdcd2a3289b818d6/)
+    // cppcheck-suppress bufferAccessOutOfBounds
```

Comment:
I think that this strxfrm is configured wrong also. The minsize for arg 2 should be removed as far as I understand. According to the documentation 2nd argument src is a null terminated string and the 3rd argument count is maximum number of characters to write.

Reply:
There is a test in `buffer_overrun_32()` which relies on the current `strxfrm` config.

Reply:
yes, as far as I see the tests below `// source size is too small` comments in buffer_overrun_32() are wrong